charts/datahub/values.yaml

```diff
@@ -111,6 +111,71 @@ datahubUpgrade:
         cpu: 300m
         memory: 256Mi
 
+prometheus-kafka-exporter:
+  enabled: false
+  kafkaServer:
+  - prerequisites-kafka:9092  # <<release-name>>-kafka:9092
+  # Sarama logging
+  sarama:
+    logEnabled: false
+  env: []
+  # - name: <ENV_VAR_NAME>
+  #   value: <value>
+
+  # List of additional cli arguments to configure kafka-exporter
+  # for example: --log.enable-sarama, --log.level=debug, etc.
+  # all the possible args can be found here: https://github.com/danielqsj/kafka_exporter#flags
+  extraArgs: []
+  
+  prometheus:
+  serviceMonitor:
```

Comment:
is indentation right here?

Reply:
Good catch, @NavinSharma13 please match indentation like this: https://github.com/prometheus-community/helm-charts/blob/88447d1886a982b0e26f047d9973d2f6bad7fe7e/charts/prometheus-kafka-exporter/values.yaml#L64

Reply:
updated in new commit